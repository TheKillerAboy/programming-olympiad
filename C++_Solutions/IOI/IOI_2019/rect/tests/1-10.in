8d9a74d5-4c4b-4437-9c49-114beaeb8f1a
30 30
1915 533 904 1917 459 602 1419 963 1587 1715 886 1351 1406 1775 1170 909 543 289 616 717 623 531 1022 1245 390 1654 23 693 539 1488
1875 743 1220 224 459 1300 755 1009 195 228 1988 103 1312 1601 1314 482 621 1919 107 1637 1477 1852 482 1402 564 986 1085 326 1 1680
655 1566 1004 812 1918 260 1570 132 445 43 1616 1237 1411 1492 591 1283 626 371 603 1033 909 1323 1873 32 1646 1672 1274 783 1256 119
960 898 1003 1865 1660 143 1164 252 222 1253 235 1491 7 1624 1403 30 1738 1613 1352 1453 240 120 216 292 142 1538 1451 458 709 1387
1072 1186 30 1974 1028 1995 565 517 1275 407 917 1286 1681 352 1445 1816 1019 1843 694 1586 201 1063 1030 1583 1993 1273 833 1197 1390 1642
1171 137 1539 59 1519 52 306 529 152 1985 1055 1207 1848 1950 128 83 1507 603 309 1409 1231 764 156 1812 1683 694 1842 1187 132 880
185 1169 1252 1376 147 599 239 1976 584 118 671 229 1112 692 1438 957 1111 1492 1536 922 365 113 105 913 1894 1646 893 1899 1214 571
1689 1745 34 110 704 1867 1815 1619 698 708 1911 1922 24 1717 1961 1473 796 1254 352 1461 3 1591 963 1783 790 735 709 1357 815 819
1620 316 513 576 512 29 270 180 811 1083 940 1696 154 1 837 887 689 47 1579 1129 1560 653 120 1538 627 1095 15 1715 1800 1431
814 636 140 509 1187 803 794 352 65 1638 249 922 1720 176 29 958 234 308 692 136 739 997 1678 1366 981 1270 1313 528 1219 459
1237 846 1110 1730 619 1662 563 205 229 1298 457 536 725 1664 1377 785 1671 1959 1465 160 587 1368 549 1707 1178 1137 1656 882 520 905
1290 156 484 1449 1806 934 1159 533 1983 87 739 319 840 1415 1916 1615 1117 211 60 412 589 550 1015 1498 971 912 632 609 597 150
802 1465 186 1332 1898 1198 452 760 1778 977 249 1380 1066 852 982 488 366 1609 1324 222 1485 1768 1182 1291 75 247 1013 1071 1543 921
1127 1768 1897 271 1397 1087 1722 294 1774 335 785 1552 180 1717 723 1644 596 636 1102 1620 59 1545 90 1274 769 145 193 1269 266 334
1376 234 182 615 1989 473 1357 962 1250 62 342 1446 1765 1251 133 1384 1969 48 1693 174 987 1070 1370 622 1600 185 433 375 659 719
960 1501 1880 1119 1557 961 1017 278 655 891 1018 1322 1812 1825 1962 961 792 1314 1017 222 1246 1930 1648 940 809 235 36 13 529 1502
74 376 385 992 32 1004 1575 1206 61 1720 1885 225 1553 1378 1408 638 293 474 1560 1679 1020 1519 1381 347 28 1432 1076 1118 570 388
1877 403 925 1935 29 1362 1279 616 879 1102 450 1255 1254 685 1345 532 100 663 1011 274 287 1425 1299 72 1551 668 649 1525 888 1577
1274 626 316 475 728 1788 1586 1150 1554 856 508 1752 1836 1055 622 1965 266 1879 236 781 946 1409 173 1502 118 833 1752 417 1414 355
782 351 1780 1750 422 1699 341 1647 1777 440 413 143 1992 1848 784 750 256 967 1005 1090 1484 741 1404 1920 1782 4 1661 786 1035 765
459 1725 389 1065 128 641 1675 909 182 645 1705 1379 1688 235 552 1250 1274 457 279 255 217 37 1998 670 1502 260 234 1828 1335 833
685 1442 1900 239 1732 290 676 873 1819 152 1310 1790 1668 1953 1633 115 744 493 880 1347 373 1547 1567 933 914 454 1671 718 337 127
102 1158 1149 329 62 21 1026 150 1334 1623 1839 1108 1052 1599 1850 255 690 1653 771 1808 660 526 687 1138 702 364 1373 1023 1439 1241
1834 728 1157 1859 503 826 609 1972 947 1447 1028 475 731 1041 1311 751 1815 783 1942 947 718 490 1390 441 1853 1063 534 482 1310 982
1096 1345 483 275 590 1639 418 1402 500 1620 1178 1211 1270 425 96 425 988 1878 1526 569 249 16 262 1512 375 1509 1982 1795 1758 1176
1387 695 423 1617 1967 471 488 1084 1558 844 320 1353 187 343 858 530 1006 655 892 1369 143 430 1957 311 323 1821 343 1380 1854 1080
1926 1200 726 1641 901 1525 1111 1195 598 433 1130 1529 392 1500 928 1765 715 1314 8 1945 952 1408 891 1155 400 724 1544 1096 1413 42
1963 1125 389 1177 843 1085 182 1583 1574 1118 1739 1139 4 1963 822 1983 1735 1916 938 243 226 1857 1565 311 472 269 1594 1721 634 1122
1809 214 267 645 462 501 1013 1041 459 1488 1150 142 368 870 1766 1128 1206 1720 198 59 684 654 1887 1464 863 1022 535 983 471 169
248 942 1881 483 843 1153 1573 401 1472 682 5 986 561 598 335 1247 1880 1509 1443 1059 926 1897 422 1760 1702 1604 500 182 350 187
