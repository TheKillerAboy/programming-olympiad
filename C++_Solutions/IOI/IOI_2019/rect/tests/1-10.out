907404fa-efbb-4a2c-83b8-4c377409c80c
OK
251
